homework01.py

```diff
@@ -0,0 +1,104 @@
+# task 01 == Виправте синтаксичні помилки.
```

Comment:
Please do no mix hometasks - one pull request -> one hometask

Reply:
I have updated the pull request, leaving only the new homework assignment (homework_03.py). Please review it again.